aries-site/src/layouts/content/DocsPageHeader.js

```diff
@@ -1,20 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Box } from 'grommet';
+import Link from 'next/link';
+import { Box, Button, Heading, PageHeader, Text } from 'grommet';
 import { getPageDetails } from '../../utils';
 import { Status, SubsectionText } from '../../components';
-import { Subsection } from '.';
 
 export const DocsPageHeader = ({ title, topic, render }) => {
   const page = getPageDetails(title);
+  const parent = topic && getPageDetails(topic);
 
   return (
-    <Subsection name={render || title} topic={topic} level={1}>
-      <Box>
-        <SubsectionText>{page.description}</SubsectionText>
-        {page.status && <Status status={page.status} />}
-      </Box>
-    </Subsection>
+    <PageHeader
+      title={
+        <Heading size="medium" margin="none" fill>
+          {render || title}
+        </Heading>
+      }
+      parent={
+        <Link href={`/${topic.toLowerCase()}`} passHref>
+          <Button>
+            <Box align="center" direction="row" gap="small">
+              {parent.icon('small', parent.color)}
+              <Text color="text">{parent.name}</Text>
```

Comment:
Do we want this to have the anchor underline treatment?

Reply:
Good question, I can reimplement as an anchor so the behavior aligns with the others examples.

Reply:
I'm actually going to wait on this piece until I demo this to Vicky. Her designs call for a button for the parent (with no underline treatment on hover like my examples). However, I was discussing with Eric if it would be sufficient to use Anchor for the parent and not have to introduce another button kind.

Reply:
Ha! I was going to chat with you on this as well. I spoke with Scott yesterday and we do want to apply anchor underline treatment on hover.